### PR TITLE
Deploy wheel to AWS; Make Mac packages be compatible with different protobuf 

### PR DESCRIPTION
**Pull request by jcwchen**
**Description**
- Deploy the built wheel to AWS to upload the wheel (upload failed somehow) manually
- Build protobuf with 3.11.3 on Mac by brew to make it support backward and forward compatibility
- Skip the failure of Python 3.5 on Mac
- Add job name on Travis-CI

**Motivation**
- Python 3.5 on Mac fails to upload PyPI package
- Mac Packages built with the latest protobuf (3.13.0) are not compatible with protobuf<3.12

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,25 @@
 env:
   global:
     - REPO_DIR=onnx
-    - BUILD_COMMIT=0c070abb0c40fec649f81a73a75b0098662ec486
+    - BUILD_COMMIT=87eb5988a5cced69b946f3d3f2963a0e271940b9
     - PLAT=x86_64
     - PB_VERSION=2.6.1
     - UNICODE_WIDTH=32
```
Comment (askhade) [inline .travis.yml]: not being used any more... can be removed

File: .travis.yml [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
+    - secure: RLmYfdQgXzHOaKVm9xq6Ao5MHXeHw9Y8YXPb0dEqPBtmMntIqcVYMKrO1h/bTAttP1LKlRMJOXS2+eiZgDzClfdvryzNgaFEXNJIY0mZ1AiraQG3UeMmJp3B+5fVLCGHCkLfqEtI5Tka46jquiUsqCajrMnHuRRQL1IPFSuPMa4lvPVSIEmBwWQIIIzfJvJrIDt5d2hVJCQ3G2Z0ylsJFQjJwbhWGT71ULpre7JWsy8XHqiFdZF3P+ELHxjdpz6XxDJkSHMZ5VIkg0XjBxAupmi8tr3vsAgRb8hfBsU/KqPRL2DLXQ0eI364+0ed4SC32aBBL1lETtiz/RLMQDdD8E62DMXLzpmqQT/dM5X3w6lWLOlCv8BAoow1iLkt8TcP2k9DXl7FCEvYagWIt2c04B3lqahdC+LM0Rx3Yc6crufWm7PzaHObFwhOzdHw3/z6J0NgvqWhVckZU4Use1bvCE4JSV8b57x/nclZRXVfu8UMAIvJAUhUs3TYOKh4V4Bgf6tD/k0K/x8Nh2qTpyEWPGq5txzJX+QEZu0SWYIULv6G3Ils9WUXplYmg77ybU/kDDoGNmjbgrdQrOPs4cdAd8iXsW1PDHBYXrbVs3mCCEk2R2o51gtVXlgWjeNYn0bXe+Ac4nHSi196ctJ4ZPLvcM5f0ccUP1wgo/h8Uo3iQWU=
```
Comment (jcwchen) [inline .travis.yml:13]: Added environment variables for AWS to get the built wheel files

**Comment by askhade** [comment]
Overall looks good. 1 comment - since mac 3.5 twine upload is failing you will have to disable twine upload (atleast for mac 3.5) and then manually upload the package

File: .travis.yml (hunk 1 above)
Comment (jcwchen) [reply .travis.yml]: I thought you are saying the BUILD_COMMIT for Windows in `appveyor.yml`? Just removed.

Comment (askhade) [reply .travis.yml]: no I meant UNICODE_WIDTH

Comment (jcwchen) [reply .travis.yml]: Does default value of UNICODE_WIDTH equal to 32? I cannot find it in the document...

File: .travis.yml [hunk 3]
```diff
@@ -145,7 +143,9 @@ script:
 
 after_success:
 - >
-  if [ -n "$TRAVIS_TAG" ]; then
+  if [ "$TRAVIS_PYTHON_VERSION" == "3.5" && "$TRAVIS_OS_NAME" == "osx"]; then
```
Comment (askhade) [inline .travis.yml]: nit: you can test this only once and add both uploading to pypi and pypi test in 1 else if loop... feel free to ignore this

**Review by askhade — APPROVED** [review]

**Review by askhade — APPROVED** [review]

**Pull request merged**